<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesWidget</class>
 <widget class="QWidget" name="SalesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raport sprzedaży</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Eksport</string>
     </property>
     <property name="icon">
      <iconset resource="guiIconsResources.qrc">
       <normaloff>:/icons/text-csv.png</normaloff>:/icons/text-csv.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dane wejściowe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="dayRadio">
        <property name="text">
         <string>Dzień</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="monthRadio">
        <property name="text">
         <string>Miesiąc</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="guiIconsResources.qrc"/>
 </resources>
 <connections/>
</ui>
